### PR TITLE
Add Intel CBnT manifests parser

### PR DESCRIPTION
**Pull request by xaionaro**
Moving from converged security suite the Intel CBnT manifests parser.

# Good

This package provides structures with parser and compiler of CBnT manifests: boot policy manifest and key manifest.

# Bad

IMHO, the most arguable thing in this package is a codegenerator (I can remove it if required). The idea is that intel structures repeats the same patters over and over, so to have a single source of truth I wrote a bit hacky [codegenerator](https://github.com/9elements/converged-security-suite/tree/master/pkg/intel/metadata/manifest/common/manifestcodegen). The main problem with it is:
* Code quality is not the best. And people complained they do not understand how it works.

The positive side about codegen is that you may remove it at any moment and just continue maintaining structures manually :)

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,8 +3,15 @@ module github.com/linuxboot/fiano
 go 1.16
 
 require (
+	github.com/dustin/go-humanize v1.0.0
+	github.com/fatih/camelcase v1.0.0
+	github.com/fatih/structtag v1.2.0 // indirect
 	github.com/spf13/pflag v1.0.5
+	github.com/stretchr/testify v1.7.0
```
Comment (rihter007) [inline go.mod:10]: I'm not sure about testify.
Fiano uses "custom" checks in tests

**Review by rihter007 — APPROVED** [review]
LGTM
The only thing that worries me is more dependencies, but this could be ok :)

File: go.mod (hunk 1 above)
Comment (xaionaro) [reply go.mod:10]: Ouh. OK. I can use standard `testing` package instead. Will fix.

Comment (orangecms) [reply go.mod:10]: I'm fine with this. If anyone complains, we can still move on. :)

**Pull request merged**